roles/identity/tasks/realm.yml

```diff
@@ -0,0 +1,41 @@
+- name: Fetch Alfresco theme
+  become: true
+  vars:
+    alfresco_theme_version: 0.3.5
+  ansible.builtin.unarchive:
+    src: "https://github.com/Alfresco/alfresco-keycloak-theme/releases/download/
+      {{ alfresco_theme_version }}/alfresco-keycloak-theme-{{ alfresco_theme_version }}.zip"
+    dest: /opt/keycloak/keycloak-{{ identity_keycloak_quarkus_version }}/themes
+    remote_src: true
+
+- name: Configure Alfresco Keycloak realm
+  community.general.keycloak_realm:
+    auth_client_id: admin-cli
+    auth_keycloak_url: http://localhost:8080
```

Comment:
Eventually that should be read from known_url but TBH I have no idea how this will all play together with the reverse proxy so good enough for I guess

Reply:
all auth_* here are connection details to reach the keycloak api run from the identity host so localhost should be good to go (port needs to be handled better but at a later stage)

Reply:
oh ok got it. I. didn't check the doc for these params tbh